oauth2_http/java/com/google/auth/oauth2/ImpersonatedCredentials.java

```diff
@@ -101,12 +103,14 @@
   private List<String> delegates;
   private List<String> scopes;
   private int lifetime;
+  private String quotaProjectId;
   private final String transportFactoryClassName;
 
   private transient HttpTransportFactory transportFactory;
 
   /**
-   * @param sourceCredentials the source credential used as to acquire the impersonated credentials
+   * @param sourceCredentials the source credential used as to acquire the impersonated credentials.
```

Comment:
delete "as",
delete final period since it's not a complete sentence

Reply:
Done